data/src/test/java/org/apache/iceberg/data/TestMetricsRowGroupFilterTypes.java

```diff
@@ -135,8 +134,17 @@
       DateTimeFormatter.ISO_LOCAL_DATE_TIME);
   private static final byte[] fixed = "abcd".getBytes(StandardCharsets.UTF_8);
 
+  @Rule
+  public TemporaryFolder temp = new TemporaryFolder();
+
+  private File orcFile;
+  private File parquetFile;
```

Comment:
Do we need two separate variables for `orcFile` and `parquetFile`?

It seems like the tests only run on one or the other at a time.

But also, I understand if you don't want to have to reorganize the tests too much. 🙂

Reply:
+1

We can probably keep a FileFormat and a File instead.